bats/fb-katello-content.bats

```diff
@@ -87,6 +110,21 @@ setup() {
     --path="/tmp/stbenjam-dummy-0.2.0.tar.gz" | grep -q "Successfully uploaded"
 }
 
+@test "upload ostree_ref" {
+  if ! tIsRHEL 8; then
+    skip "OSTree content is only applicable on EL 8 systems"
```

Comment:
Should we be forward compatible and assume it will also be on EL 9 systems?
```suggestion
  if tIsRHEL 7; then
    skip "OSTree content is only not applicable on EL 7 systems"
```

(repeated elsewhere, not commenting on it everywhere)